tox.ini

```diff
@@ -4,15 +4,7 @@ envlist = qa, static, docs, py{37,38,39,310,311,py3}-{nocov,cov,diffcov}
 skip_missing_interpreters = True
 
 [testenv]
-# One virtualenv per Python version
-envdir =
-    py37: {toxworkdir}/3.7
-    py38: {toxworkdir}/3.8
-    py39: {toxworkdir}/3.9
-    py310: {toxworkdir}/3.10
-    py311: {toxworkdir}/3.11
-    pypy3: {toxworkdir}/pypy3
-    py: {toxworkdir}/py
```

Comment:
Previously, if you specify something not listed here (e.g., `pypy38-nocov`), tox gets a bit confused and will decide to wipe out the whole cloned repo and creates its dirs in your repo's location. It also wipes recursively the contents of `.git`, so you can't do a `reset --hard`.

Reply:
Oof. Well, that find sucks but glad you were able to recover :+1:

Reply:
Thankfully there was no unpushed commits when that happened, so it's a quick git clone and I recovered.